.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+__pycache__
+*s3-bucket/.terraform
+*s3-bucket/.terraform.lock.hcl
```

Comment:
The .gitignore i shared won't include `.terraform.lock.hcl` but keep it anyways as we usually don't keep the lock files for our terraform. Instead do add a file that has this module: https://github.com/GlueOps/terraform-module-provider-versions

This let's us keep our entire stack on the same terraform provider versions.